### ISSUE TITLE
"Is Anchor" option is missing from categories?

### ISSUE THREAD

**Issue by octoxan** [Smile-SA/smile-magento-elasticsearch#60]
After installing this extension, there is no longer an option in Manage Categories for "Is Anchor". Things that were an anchor before are still an anchor, but there is no longer a way to set categories to be or not to be anchors.

**Comment by romainruaud** [comment]
You are right, according to the setup file (https://github.com/Smile-SA/smile-magento-elasticsearch/blob/master/src/app/code/community/Smile/ElasticSearch/sql/smile_elasticsearch_setup/upgrade-1.1.0-1.1.1.php), all future categories are considered as being anchors (as the field still exists but is now an hidden input field which defaults to 1).

**Comment by octoxan** [comment]
I see.. what about for previously existing categories?

Before installing, one category had Is Anchor set to No and one had it set to Yes. The one previously set to No still shows no attributes in the filter (besides price and subcategory). Reindexed and flushed cache. If I inspect the hidden input, it says its value is 1.. however no attributes show up for the category still.

**Comment by romainruaud** [comment]
You are right, previously existing categories may remain at "No" state after running the install script. However, since the default value for this field is now 1, did you try to save this particular category to force the flag to be set to 1 in database ?

**Comment by octoxan** [comment]
Just saving the category makes the hidden input remain value="0", but editing the hidden input with dev tools to make it a "1" and then saving it makes the website thinks its an anchor.. but it still doesn't function properly as one. No filters actually show up still but a line of code we have to tell us if its an anchor shows up.. just nothing to filter by. Products in this category are the same as the one that was set to an anchor before installation, and that other category has things to filter by.

**Comment by octoxan** [comment]
Filters show up after narrowing down the list of products using the price slider to at least 3 products...

**Comment by amenk** [comment]
@romainruaud so is it by design, that when SmileSA module is in use, categories cannot be non-anchor?

**Comment by romainruaud** [comment]
Exactly.

**Comment by amenk** [comment]
@romainruaud Whoops .. did not know that. Actually I am talking about the Magento 2 version
https://github.com/Smile-SA/elasticsuite/ Can you point me to some relevant issues or doc?
Is there a replacement for those? In our current shop-design we currently depend on some non-anchors
